nvflare/apis/workspace.py

```diff
@@ -18,34 +18,110 @@
 
 
 class Workspace:
-    def __init__(self, root_dir: str, name: str, config_folder: str):
+    def __init__(self, root_dir: str, site_name: str = "", config_folder: str = "config"):
         """Define a workspace.
 
         NOTE::
 
             Workspace folder structure:
 
                 Workspace ROOT
+                    local
+                        authorization.json.default
+                        resources.json.default
+                        ...
                     startup (optional)
                         provisioned content
+                        fed_client.json
```

Comment:
this workspace is also on server side, so we don't want to have fed_client.json here?

Reply:
Then we should add description saying "The workspace on client site can look like:"

Reply:
updated.